Sample Applications/WPFGallery/ViewModels/Samples/UserDashboardPageViewModel.cs

```diff
@@ -42,40 +40,56 @@ private void AddUser()
         {
             Users.Add(new User("New User", ""));
             SelectedUser = Users.Last();
-            EditableUser = new User(SelectedUser);
```

Comment:
Why is this removed ? When we are adding a new user, the page should be in editable mode.

Reply:
When you add a new user, the `OnSelectedUserChanged` method is fired which has the same logic. This assignment was redundant.